database/things/uxf.pldb

```diff
@@ -11,13 +11,28 @@ originCommunity Qtrac Ltd
 domainName qtrac.eu
 latestRelease 2022
 
+features
+ hasBooleans true
+ hasComments true
+  # <This is a comment>
+ hasStrings true
+  <Hello world>
```

Comment:
Interesting! They do strings a bit differently eh?

Reply:
yes, they  use <> for strings